src/common/GPOWrapper/GPOWrapper.cpp

```diff
@@ -144,4 +144,8 @@ namespace winrt::PowerToys::GPOWrapper::implementation
     {
         return static_cast<GpoRuleConfigured>(powertoys_gpo::getAllowExperimentationValue());
     }
+    GpoRuleConfigured GPOWrapper::GetConfiguredEnvironmentVariablesEnabledValue()
+    {
+        return static_cast<GpoRuleConfigured>(powertoys_gpo::getConfiguredEnvironmentVariablesEnabledValue());
```

Comment:
If #28731 gets merged the way of implementing the utility policies changes a bit. Please be aware of this change.